grape/utils/vertex_array.h

```diff
@@ -322,6 +322,10 @@ class DualVertexRange {
     return VertexRange<VID_T>(tail_begin_, tail_end_);
   }
 
+  const VID_T begin_value() const { return head_begin_; }
+
+  const VID_T end_value() const { return tail_end_; }
+
```

Comment:
Why did you need to expose those two internal fields? Looks not used.

Reply:
Use in [sssp](https://github.com/alibaba/libgrape-lite/pull/127/files#diff-310e5cbf407d7b968e25456e8caf45cc692e03d783287ced1f607774eb439c74L158) algorithm with mutable fragment (DualVertexRange)